Trigger CI using Terraform Cloud Notification

kameshsampath opened the pull request:
how to use terraform cloud notification to trigger CI pipeline

bot-gitexp-user commented:
Please check the Execution Link of the Pipeline for the Website Draft URL. This is located in the Preview Step and also is available in #hdh_alerts. E.g Website Draft URL: https://unique-id--harness-developer.netlify.app

ravilach commented on src/components/LandingPage/data/allTutorialsData.tsx:
```diff
@@ -61,6 +61,20 @@ export const FeaturedList: CardItem[] = [
+    newDoc: true,
+    type: [docType.Documentation],
+    time: '9 min',
+    link: "/tutorials/build-code/ci-tfc-notification-trigger",
```
Link is not valid. would be under /build-code/build

ravilach reviewed (approved):
Not sure why there was a MC here: https://developer.harness.io/tutorials/build-code/build/

aimurphy commented:
@kameshsampath For `Add harnessImage Docker Registry Connector`, is there a specific reason you need to [override the default Harness image connector (for the Initialize step)](https://developer.harness.io/docs/platform/connectors/connect-to-harness-container-image-registry-using-docker-connector/) or can it be [any DockerHub connector](https://developer.harness.io/docs/platform/connectors/ref-cloud-providers/docker-registry-connector-settings-reference/)?

aimurphy commented:
@kameshsampath This repo doesn't exist: https://github.com/harness-apps/bootstrap-gke

bot-gitexp-user commented:
Please check the Execution Link of the Pipeline for the Website Draft URL. This is located in the Preview Step and also is available in #hdh_alerts. E.g Website Draft URL: https://unique-id--harness-developer.netlify.app

aimurphy commented:
Preview link: https://6435d20010fd24008bb4162b--harness-developer.netlify.app/tutorials/build-code/build/tfc-notification

kameshsampath commented:
> @kameshsampath For `Add harnessImage Docker Registry Connector`, is there a specific reason you need to [override the default Harness image connector (for the Initialize step)](https://developer.harness.io/docs/platform/connectors/connect-to-harness-container-image-registry-using-docker-connector/) or can it be [any DockerHub connector](https://developer.harness.io/docs/platform/connectors/ref-cloud-providers/docker-registry-connector-settings-reference/)?

I need to use `harnessImage` as I want to user to use `anonymous` pulls

kameshsampath commented:
> @kameshsampath This repo doesn't exist: https://github.com/harness-apps/bootstrap-gke
> 
> I think it's supposed to be the bootstrap-argocd repo.
good catch and updated.
> 
> Also I think the user needs to clone the [tfc-notification-demo repo](https://github.com/harness-apps/tfc-notification-demo) so they can get the pipeline yaml to import?
great! added those instructions too.

bot-gitexp-user commented:
Please check the Execution Link of the Pipeline for the Website Draft URL. This is located in the Preview Step and also is available in #hdh_alerts. E.g Website Draft URL: https://unique-id--harness-developer.netlify.app

aimurphy commented:
close due to impact of force push. Will remake new PR from my local
